modules/rtm/inst/scripts/testing_foursail.R

```diff
@@ -0,0 +1,35 @@
+library(PEcAnRTM)
+
+# get soil
+rsoil <- system.file("extdata", "soil_reflect_par.dat", package="PEcAnRTM")
+rsoil <- read.table(rsoil,header = F)
+#str(rsoil)
+rsoil <- as.vector(unlist(rsoil[1,]))
+length(rsoil)
+rsoil <- c(rsoil,rsoil[2100]) # make soil reflectance the correct length
+plot(seq(400,2500,1), rsoil, type = "l")
+
+# define some sail params
+LIDFa <- -0.35 
+LIDFb <- -0.15
+TypeLIDF <- 1
+LAI <- 4
+q <- 0.1
+tts <- 48
+tto <- 0 
+psi <- 234
+params <- c(LIDFa,LIDFb,TypeLIDF,LAI,q,tts,tto,psi)
+param <- params
+
+# get leaf refl/trans
+LRT <- PEcAnRTM::prospect(c(2,55,10,3,0.1,0.007,0.007), 'D')
+plot(LRT[,1])
+refl <- LRT[,1]
+length(refl)
+tran <- LRT[,2]
+
+# generate 4SAIL canopy spectra
+sail_spec <- foursail(refl, tran, rsoil, params)
```

Comment:
It would be nice to convert this to a unit test. You could basically keep all of the code, with the following changes:
* Drop the `library` call.
* Add a `context("SAIL canopy RTM with custom inputs")
* Add one or two very simple `expect_*` statements, such as `expect_equal(nrow(sail_spec), 2101); expect_equal(ncol(sail_spec), 4); expect_true(all(sail_spec) > 0); expect_true(all(is.finite(sail_spec)))`

Reply:
I can take care of this. I just pulled the PR and can add a patch.

Reply:
OK thats cool @ashiklom thanks!

Reply:
Updated with new test [that you prepared for me :) ]